main.py

```diff
@@ -2,6 +2,7 @@
 from prompt_toolkit import print_formatted_text, HTML
 from websockets.exceptions import ConnectionClosedError
 from utils.helper_display import HelperDisplay
+from requests import get
```

Comment:
`requests` is quite a heavy module to import for task as small as this. Also, we cannot just assume that the users have `curl` or `wget` binaries installed on their distribution which makes it a difficult point to discuss about.